src/main/java/uk/gov/ch/repository/transaction/Gaz2TransactionRepository.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.ch.repository.transaction;
+
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.PagingAndSortingRepository;
+import uk.gov.ch.model.transaction.sqldatamodels.Gaz2TransactionDataModel;
+
+public interface Gaz2TransactionRepository extends PagingAndSortingRepository<Gaz2TransactionDataModel, Long> {
+    @Query(value = "select tr.TRANSACTION_ID, tr.TRANSACTION_TYPE_ID, tr.TRANSACTION_STATUS_TYPE_ID " +
+            "from corporate_body cb, transaction tr " +
+            "where tr.CORPORATE_BODY_ID = cb.CORPORATE_BODY_ID " +
+            "and tr.TRANSACTION_TYPE_ID in (541, 542) " +
+            "and tr.TRANSACTION_STATUS_TYPE_ID = 26 " +
+            "and cb.INCORPORATION_NUMBER = ?1",
```

Comment:
I would go with a named param here, easier to read and less error prone - https://www.baeldung.com/spring-data-jpa-query#named-parameters

Reply:
could there be a resultSet returned with size > 1? 
If a company has been dissolved several times, what does it's transaction history look like? I thought each time a change was made to a company a new transaction was created which might mean there could be > 1 gaz2 with status of 26. Might be worth checking with a dba (mikee)

Reply:
From Sarah:
"Connie/Dylan said you can have multiple issued but not requested."

So searching by requested should either return 0 or 1